summingbird-storm-test/src/test/scala/com/twitter/summingbird/storm/StormLaws.scala

```diff
@@ -99,72 +47,57 @@ object StormLaws {
   }
 
   val serviceFn = sample[Int => Option[Int]]
-  val service = ReadableServiceFactory[Int, Int](() => ReadableStore.fromFn(serviceFn))
-
 }
 
 // ALL TESTS START GO IN THE CLASS NOT OBJECT
 
 class StormLaws extends WordSpec {
   import StormLaws._
-  import MapAlgebra.sparseEquiv
 
   "StormPlatform matches Scala for single step jobs" in {
     val original = sample[List[Int]]
-    val returnedState =
-      StormTestRun.simpleRun[Int, Int, Int](original,
-        TestGraphs.singleStepJob[Storm, Int, Int, Int](_, _)(testFn)
-      )
-
-    assertEquiv[Map[Int, Int]](
-      TestGraphs.singleStepInScala(original)(testFn),
-      returnedState.toScala
-    )
+    val fn = sample[Int => List[(Int, Int)]]
+
+    testStormEqualToMemory(new ProducerCreator {
```

Comment:
Can we create a helper method to simplify this code for readability:
testStormEqualToMemory(new ProducerCreator { def apply = fn })
to
testStormEqualToMemory(fn)